lightning/src/onion_message/functional_tests.rs

```diff
@@ -111,16 +111,37 @@ impl Writeable for TestCustomMessage {
 }
 
 struct TestCustomMessageHandler {
-	expected_messages: Mutex<VecDeque<TestCustomMessage>>,
+	expected_messages: Mutex<VecDeque<Expectations>>,
+}
+
+struct Expectations {
+	message: TestCustomMessage,
+	add_reply_path: bool,
 }
```

Comment:
How about this?

```rust
struct OnHandleCustomMessage {
	expect: TestCustomMessage,
	respond_with: Option<TestCustomMessage>,
	include_reply_path: bool,
}
```

Probably don't need `respond_with` now, but just wanted to show how this could be expanded in the future if needed.

Reply:
I think the naming sounds better, as it makes it more expressive in what it does.
Took up your suggestion in [e6effd4](https://github.com/lightningdevkit/rust-lightning/commit/e6effd447d47e1c3c7771935bea58801199b60b3)
Thanks!